### PR TITLE
8335684: Test ThreadCpuTime.java should pause like ThreadCpuTimeArray.java

### PR DESCRIPTION
**Pull request by kevinjwalls**
There are two similarly names tests.
Recently:
JDK-8335124: com/sun/management/ThreadMXBean/ThreadCpuTimeArray.java failed with CPU time out of expected range
...made a simple change to try and avoid noisy test failures.  The same fix should be applied here to ThreadCpuTime.java.

Also removing an old comment about a Solaris issue.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8335684](https://bugs.openjdk.org/browse/JDK-8335684): Test ThreadCpuTime.java should pause like ThreadCpuTimeArray.java (**Bug** - P4)

### Reviewers
 * [Serguei Spitsyn](https://openjdk.org/census#sspitsyn) (@sspitsyn - **Reviewer**)
 * [Chris Plummer](https://openjdk.org/census#cjplummer) (@plummercj - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/20025/head:pull/20025` \
`$ git checkout pull/20025`

Update a local copy of the PR: \
`$ git checkout pull/20025` \
`$ git pull https://git.openjdk.org/jdk.git pull/20025/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 20025`

View PR using the GUI difftool: \
`$ git pr show -t 20025`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/20025.diff">https://git.openjdk.org/jdk/pull/20025.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/20025#issuecomment-2208713400)

### CODE REVIEW

**Review by sspitsyn — APPROVED** [review]
Okay to me.

File: test/jdk/java/lang/management/ThreadMXBean/ThreadCpuTime.java [hunk 1]
```diff
@@ -228,15 +230,6 @@ public void run() {
...
             }
-/*************
- * FIXME: Seems that on Solaris-sparc,
- * It occasionally returns a different current thread user time > thread user time
-            if (utime1 > utime2) {
-                throw new RuntimeException("TEST FAILED: " + getName() +
-                    " CurrentThreadUserTime = " + utime1 +
-                    " > ThreadUserTime = " + utime2);
-            }
-*/
```
Comment (plummercj) [inline test/jdk/java/lang/management/ThreadMXBean/ThreadCpuTime.java:239]: Shouldn't this be uncommented and this bit of testing restored? It seems the only reason it was commented out is because of Solaris, which we don't need to worry about anymore. Probably best to leave this to a separate PR if you are going to restore it.

Comment (kevinjwalls) [reply test/jdk/java/lang/management/ThreadMXBean/ThreadCpuTime.java:239]: Would have been happy to test it and bring it back, have looked into it more:

back in jdk5u we have the same commented out code, we have never run this in general testing. 

I think it's redundant, the two calls are equivalent.

  long utime1 = mbean.getCurrentThreadUserTime();

"This is a convenient method for local management use and is equivalent to calling: getThreadUserTime(Thread.currentThread().getId());"

..which is the other time we would be comparing it with:

  long utime2 = mbean.getThreadUserTime(getId()); (in this class that extends Thread).
  
I think we can presume some os-specific quirk that does not affect us today!

Comment (plummercj) [reply test/jdk/java/lang/management/ThreadMXBean/ThreadCpuTime.java:239]: ok.

**Review by plummercj — APPROVED** [review]
Looks good.

**Comment by kevinjwalls** [comment]
thanks for the reviews!
